### PR TITLE
Feature/wercker build

### PR DESCRIPTION
**Pull request by antnieszka**
Improvements in #82

### CODE REVIEW

**Comment by antnieszka** [comment]
@grzanka This PR covers more than 50% of list from #105 (most important) - maybe we should merge it and in free time I'll finish?

File: wercker.yml [hunk 1]
```diff
@@ -1,74 +1,829 @@
...
+        code: |
+          apt-get update
+          apt-get install -qq -y --no-install-recommends python-pip python3-pip python-numpy python3-numpy
+          python2 -V
+          pip --version
+          python3 -V
+          pip --version
+
+          easy_install -U pip
+          easy_install3 -U pip==7.1.2
```
Comment (grzanka) [inline wercker.yml]: I'd rather recommend using "pip<8.0.0" than a fixed version. Fixed version might disappear, using "newest which is older than 8.0.0" is safer.

File: wercker.yml [hunk 2]
```diff
@@ -1,74 +1,829 @@
...
+  steps:
+    - script:
+        name: prerequisites
+        code: |
+          apt-get update
+          apt-get install -qq -y --no-install-recommends python python3 python-setuptools python3-setuptools python-numpy python3-numpy
+          python2 -V
+          python3 -V
+          easy_install -U pip
+          easy_install3 -U pip==7.1.2
```
Comment (grzanka) [inline wercker.yml]: Same as above

File: wercker.yml [hunk 3]
```diff
@@ -1,74 +1,829 @@
...
+          python3 -V
+          pip2 install --upgrade pip
+          pip2 -V
+          pip3 install --upgrade pip
+          pip3 -V
+
+    - script:
+        name: install numpy from repo
+        code: |
+          zypper --non-interactive in python-numpy python3-numpy
```
Comment (grzanka) [inline wercker.yml]: better to write explicitely "install" instead of "in"

File: wercker.yml [hunk 4]
```diff
@@ -1,74 +1,829 @@
...
+# Archlinux
+
+test_archlinux_repo:
+  box: base/archlinux:latest
+
+  steps:
+    - script:
+        name: prerequisites
+        code: |
+          echo "Server = ftp://ftp.icm.edu.pl/pub/Linux/dist/archlinux/core/os/x86_64/" > /etc/pacman.d/mirrorlist
```
Comment (grzanka) [inline wercker.yml]: Why is that needed ? Add some comment explaining this strange echo.
I am not using official base/archlinux as it looks abandoned.
Instead there is finalduty/archlinux docker. 

See also https://github.com/grzanka/python-pip-versions/blob/master/wercker.yml

I have also my own docker images with preinstalled python. Dockerfiles are here:
https://github.com/grzanka/python-pip-versions/tree/master/Dockerfiles
Images are automatically built here: https://hub.docker.com/r/grzanka/

I am not sure if they can be of some profit or not as wercker installs python really fast.

You can also check files other than README.md here: https://github.com/grzanka/python-pip-versions

Comment (antnieszka) [reply wercker.yml]: Well - I've changed docker image to finalduty/archlinux and it works without `echo ...`. Probably the base/arch was broken or something (at least mirrors were).

**Comment by grzanka** [comment]
Can you add an issue writing down what is missing in terms of wercker config (like proper testing etc) ?

**Pull request merged**